salt/modules/status.py

```diff
@@ -1054,8 +1054,12 @@ def master(master=None, connected=True):
     if not master_ips:
         return
 
+    pub_port = None
```

Comment:
its not imported. 
dunno how you do that for the rest but what I can do it something like this -

```
def connected_masters():
    '''
    Return current connected masters
    '''
    # default port
    port = 4505

    import salt.config
    import salt.loader

    __opts__ = salt.config.minion_config('/etc/salt/minion')
    configmod = salt.loader.raw_mod(__opts__, 'config', None)
    config_port = configmod['config.get']('publish_port')
    # etc .......
```

ofc I can do the imports at the top or whatever you think its best.

I do think that there is no reason to go into the trouble for this since it can just be passed from the module. None of the utils has any calls to modules anyway.

Let me know what you think.